### DISCUSSION TITLE
How to save to PDF?

### DISCUSSION THREAD

**Discussion by zzador** [vkbo/novelWriter#1900] [Q&A] [answered]
Hello everybody,

In the Features-List on the homepage of novelWriter it says novelWriter can export to PDF. How can I generate a PDF? Building a manuscript doesn't support PDF output format.

**Comment by vkbo** [comment, marked as answer]
https://novelwriter.io/docs/project_manuscript.html#print-and-pdf

**Reply by vkbo** [reply]
I've clarified the feature text. The Manuscript docs are linked from under that feature box though.
